### DISCUSSION TITLE
Domain Expired due to wrong card details

### DISCUSSION THREAD

**Discussion by ismailelbadawy** [vercel/community#7369] [Help]
### Summary

I have a team called **Icteam** and i had a domain `ghoniemonline.com` and it has expired due to my card rejecting the transaction. 

The domain is out of reach as of today. I need to get it back up as soon as possible. 

I tried opening a case but no one is replying on the support nor on the email `billing@vercel.com`.

I need some help please as the services are unreachable due to the domain being unreachable. 

### Steps to Reproduce

On the domain page there is no way to renew the domain. as you can see below
<img width="1450" alt="Screenshot 2024-07-04 at 9 15 01 PM" src="https://github.com/vercel/community/assets/36375277/118d5e09-7381-41e9-a54c-ab9b58539c6d">

**Comment by ismailelbadawy** [comment]
@amyegan can you please help, I can see you have replied to similar problems on the github discussions before can you somehow inform the team of my open case (case number 00254428).

I need help with this very much and I don't know how to reach support because you guys do not respond to my open cases. The probability of me losing a client is increasing by the minute. I just want to renew my domain.

**Reply by pawlean** [reply]
Hi @ismailelbadawy, 

I've shared this internally, and will get back to you as soon as we can! Thanks for your patience.

**Reply by pawlean** [reply]
@ismailelbadawy This should now be resolved. Please let us know if we can help with anything else!

[a comment by an automated account is not shown]
